.eslintrc.json

```diff
@@ -64,6 +58,28 @@
     "react/jsx-uses-react": "off",
     "react/react-in-jsx-scope": "off",
     "no-irregular-whitespace": "off",
-    "@next/next/no-img-element": "off"
+    "@next/next/no-img-element": "off",
+
+    // turn on errors for missing imports
+    "import/no-unresolved": "error",
+    "import/order": [
+      "error",
+      {
+        "groups": [
+          "builtin", // Built-in imports (come from NodeJS native) go first
+          "external", // <- External imports
+          "internal", // <- Absolute imports
+          ["sibling", "parent"], // <- Relative imports, the sibling and parent types they can be mingled together
```

Comment:
`Relative imports, the sibling and parent types they can be mingled together`
`Relative imports, the sibling and parent types can be mingled together`

Reply:
J'avais repris direct le bout de code d'exemple, mais je vais corriger la phrase tant qu'à faire. :+1: